.circleci/config.yml

```diff
@@ -2,34 +2,25 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: cimg/node:16.4.0
+      - image: cimg/node:18.15.0
 
     working_directory: ~/dom-query
 
     steps:
       - checkout
 
-      - restore_cache:
-          key: dependency-cache-{{ checksum "yarn.lock" }}
-
       - run:
           name: Install Javascript dependencies
-          command: yarn install --pure-lockfile
-
-      - save_cache:
-          key: dependency-cache-{{ checksum "yarn.lock" }}
-          paths:
-            - ~/.cache/yarn
-            - node_modules
```

Comment:
This packages doesn't have a lot of dependencies to install and pnpm is blazing fast so it's slower and not necessary to save/restore cache here